src/mainmenu.asm

```diff
@@ -818,6 +818,8 @@ ih_display_mode:
     db #$28, " X POSITION", #$FF
     db #$28, " Y POSITION", #$FF
     db #$28, "   COOLDOWN", #$FF
+    db #$28, " SHINE TUNE", #$FF
+    db #$28, " SHOT TIMER", #$FF
```

Comment:
I did find the code that allows the user to scroll left or right in the menu, and it already had support for 12 items in the list even though there were only 10, so this time I could just add items here.  Next time we'll have to update the menu scrolling logic.